libraries/chain/db_maint.cpp

```diff
@@ -1059,88 +1051,12 @@ void process_hf_868_890( database& db, bool skip_check_call_orders )
          }
       } // end loop of each feed
 
-      // if any feed was modified, print a warning message
-      if( feeds_changed )
-      {
-         wlog( "Found invalid feed for asset ${asset_sym} (${asset_id}) during hardfork core-868-890",
-               ("asset_sym", current_asset.symbol)("asset_id", current_asset.id) );
-      }
-
       // always update the median feed due to https://github.com/bitshares/bitshares-core/issues/890
       db.modify( bitasset_data, [head_time,next_maint_time]( asset_bitasset_data_object &obj ) {
          obj.update_median_feeds( head_time, next_maint_time );
       });
 
```

Comment:
Please add a note after `update_median_feeds`: normally we should call `check_call_orders()` after called `update_median_feeds()`, but for mainnet actually `check_call_orders()` would do nothing, so we skipped it for better performance.